tests/gui/components/drive-selector.spec.js

```diff
@@ -20,71 +20,17 @@ const _ = require('lodash')
 const m = require('mochainon')
 const angular = require('angular')
 require('angular-mocks')
+const utils = require('../../../lib/shared/utils')
 
 describe('Browser: DriveSelector', function () {
   beforeEach(angular.mock.module(
     require('../../../lib/gui/app/components/drive-selector/drive-selector')
   ))
 
   describe('DriveSelectorController', function () {
-    let $controller
-    let $rootScope
-    let $q
-    let $uibModalInstance
-    let WarningModalService
-
-    let controller
-
-    beforeEach(angular.mock.inject(function (
-      _$controller_,
-      _$rootScope_,
-      _$q_,
-      _WarningModalService_
-    ) {
-      $controller = _$controller_
-      $rootScope = _$rootScope_
-      $q = _$q_
-      $uibModalInstance = {}
-      WarningModalService = _WarningModalService_
-    }))
-
-    beforeEach(() => {
-      controller = $controller('DriveSelectorController', {
-        $scope: $rootScope.$new(),
-        $q,
-        $uibModalInstance,
-        WarningModalService
-      })
-    })
-
-    describe('.memoizeImmutableListReference()', function () {
-      it('constant true should return memoized true', function () {
-        const memoizedConstTrue = controller.memoizeImmutableListReference(_.constant(true))
-        m.chai.expect(memoizedConstTrue()).to.be.true
-      })
-
-      it('should reflect state changes', function () {
-        let stateA = false
-        const memoizedStateA = controller.memoizeImmutableListReference(() => {
-          return stateA
-        })
-
-        m.chai.expect(memoizedStateA()).to.be.false
-
-        stateA = true
-
-        m.chai.expect(memoizedStateA()).to.be.true
-      })
-
-      it('should reflect different arguments', function () {
-        const memoizedParameter = controller.memoizeImmutableListReference(_.identity)
-
-        m.chai.expect(memoizedParameter(false)).to.be.false
-        m.chai.expect(memoizedParameter(true)).to.be.true
-      })
-
+    describe('.memoize()', function () {
```

Comment:
I don't fully understand the code in this PR, but why is this test in `drive-selector.spec.js` rather than `utils.spec.js` ? AFAICT there's nothing DriveController-specific about it?

Reply:
The other isn't a GUI test so Angular isn't available (because `window` isn't), since we use `.memoize()` in the DriveSelectorController in an Angular-specific way (`angular.equals`) I think it makes most sense to keep it here.

Reply:
Ahhh, that makes sense - I hadn't noticed that the test here was using `angular.equals` but the tests in `utils.spec.js` were using `_.isEqual` :)  (as you can probably tell, I know nothing about Angular)